user/timepix3/module/src/Timepix3Event2StdEventConverter.cc

```diff
@@ -82,14 +83,50 @@ uint64_t Timepix3RawEvent2StdEventConverter::m_syncTime(0);
 uint64_t Timepix3RawEvent2StdEventConverter::m_syncTime_prev(0);
 size_t Timepix3RawEvent2StdEventConverter::m_clearedHeader(0);
 bool Timepix3RawEvent2StdEventConverter::m_first_time(true);
-bool Timepix3RawEvent2StdEventConverter::Converting(eudaq::EventSPC ev, eudaq::StandardEventSP d2, eudaq::ConfigurationSPC conf) const{
+bool Timepix3RawEvent2StdEventConverter::applyCalibration(false);
+std::string Timepix3RawEvent2StdEventConverter::calibrationPathToT("");
```

Comment:
Again, why do you need members here? These files are read exactly once and the path is not needed afterwards.

Reply:
You're totally right, I changed it now.